docs/CryptolPrims.md

```diff
@@ -53,6 +71,10 @@ Boolean
     (^)   : a -> a -> a
     (~)   : a -> a
 
+    (==>) : Bit -> Bit -> Bit
+    (/\)  : Bit -> Bit -> Bit
+    (\/)  : Bit -> Bit -> Bit
+
```

Comment:
It's true, they're a bit unrelated.  The document had fallen a bit out of date in general.

I can split it into separate patches, if you think it's important.